platform/core/src/DICOMSR/utils/findMostRecentStructuredReport.js

```diff
@@ -37,6 +37,10 @@ const isStructuredReportSeries = series => {
   ];
 
   const firstInstance = series.getFirstInstance();
+  if (!firstInstance) {
+    return false;
+  }
```

Comment:
Is this a timing issue caused by these changes? Are we checking "too early" and this short-circuits a would-be exception? Do we eventually re-check to grab the appropriate SR?

If this is used somewhere that's reactive, would the check be more appropriate one layer up?

Reply:
I'm not actually sure. I had this in as a fix at one point but it might not be necessary any longer.

Reply:
Ah it's just to prevent this from failing if a series has no instances yet. I moved it one level up.